README.md

```diff
@@ -1,211 +1,2 @@
-both through command line arguments and interactively.
-```bash
-python scripts/nlp/intentslot_client.py --query "What is the weather tomorrow?"
-```
-or
-```bash
-python scripts/nlp/intentslot_client.py --interactive
-```
-For punctuation client the commands look similar.
-```bash
-python scripts/nlp/punctuation_client.py --query "can you prove that you are self aware"
-```
-or
-```bash
-python scripts/nlp/punctuation_client.py --interactive
-```
-
-**NER** client can output 1 of the following: label name, span start, span end
-```bash
-python scripts/nlp/ner_client.py \
-  --query "Where is San Francisco?" "Jensen Huang is the CEO of NVIDIA Corporation." \
-  --test label
-```
-or
-```bash
-python scripts/nlp/ner_client.py \
-  --query "Where is San Francisco?" "Jensen Huang is the CEO of NVIDIA Corporation." \
-  --test span_start
-```
-or
-```bash
-python scripts/nlp/ner_client.py \
-  --query "Where is San Francisco?" "Jensen Huang is the CEO of NVIDIA Corporation." \
-  --test span_end
-```
-
-Provide query and context to **QA** client.
-```bash
-python scripts/nlp/qa_client.py \
-  --query "How many gigatons of carbon dioxide was released in 2005?" \
-  --context "In 2010 the Amazon rainforest experienced another severe drought, in some ways "\
-"more extreme than the 2005 drought. The affected region was approximate 1,160,000 square "\
-"miles (3,000,000 km2) of rainforest, compared to 734,000 square miles (1,900,000 km2) in "\
-"2005. The 2010 drought had three epicenters where vegetation died off, whereas in 2005 the "\
-"drought was focused on the southwestern part. The findings were published in the journal "\
-"Science. In a typical year the Amazon absorbs 1.5 gigatons of carbon dioxide; during 2005 "\
-"instead 5 gigatons were released and in 2010 8 gigatons were released."
-```
-
-**Text classification** requires only a query.
-```bash
-python scripts/nlp/text_classify_client.py --query "How much sun does california get?"
-```
-
-#### TTS
-
-Call ``scripts/tts/talk.py`` script, and you will be prompted to enter a text for speech
-synthesis. Set `--play-audio` option, and a synthesized speech will be played.
-```bash
-python scripts/tts/talk.py --play-audio
-```
-
-You can write output to file.
-```bash
-python scripts/tts/talk.py --output 'my_synth_speech.wav'
-```
-
-You can use streaming mode (audio fragments returned to client as soon as they are ready).
-```bash
-python scripts/tts/talk.py --stream --play-audio
-```
-
-### API
-
-See tutorial notebooks in directory `tutorials`.
-
-
-## Documentation
-
-Additional documentation on the Riva Speech Skills SDK can be found [here](https://docs.nvidia.com/deeplearning/riva/user-guide/docs/).
-
-
-## License
-
-This client code is MIT-licensed. See LICENSE file for full details.
+Use `nvidia-riva-client <https://github.com/nvidia-riva/python-clients>`_ instead.
+`riva-api` package is deprecated.
```

Comment:
Not sure what syntax pypi uses for links, but this would be for regular Markdown

```suggestion
Use [`nvidia-riva-client`](https://github.com/nvidia-riva/python-clients) instead.
`riva-api` package is deprecated.
```

Reply:
You can manage syntax by `long_description_content_type` parameter. Default is RST. I wanted to reduce amount of parameters and removed `long_description_content_type="text/markdown"` from `setup()` call. It looks like description is rendered correctly https://test.pypi.org/project/riva-api/.

Reply:
Thanks for the info! Looks good

Reply:
Updating the link to regular markdown may be beneficial for those who end up on github, but the majority of folks  will probably visit the pypi first.